lxqt-admin-user/lxqt-admin-user-helper.freebsd

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+case "$1" in
+useradd|usermod|userdel|groupadd|groupmod|groupdel)
+    export LC_ALL=C
+    PW=`which pw`
```

Comment:
Does `which` on BSD do something different than (doc): 
```
which returns the pathnames of the files (or links) which would be executed in the current environment, had its arguments been given as commands in a strictly POSIX-
conformant shell.  It does this by searching the PATH for executable files matching the names of the arguments. It does not canonicalize path names.
```
?

If not, I believe, this is not neccessary.